src/packageJson.ts

```diff
@@ -9,10 +9,10 @@ export async function copyPackageJsonFile(cwd: string, outDir: string) {
   executeCmd(`cp ${pjsonPath} ${outDir}`);
 }
 
-export async function rewritePackageJsonFile(cwd: string, outDir: string) {
+export async function rewritePackageJsonFile(cwd: string, outDir: string, noCJS: boolean) {
   const pjsonDistPath = path.join(outDir, 'package.json');
   const pjson = JSON.parse(await fs.readFile(pjsonDistPath, 'utf8')) as PackageJson;
-  pjson.main = './index.cjs.js';
+  if (!noCJS) pjson.main = './index.cjs.js';
```

Comment:
@pbeshai should this be rewritten to the ESM entry point if CJS is disabled?

Reply:
I'm not entirely sure. I feel like some time along with my testing having a main at all was causing issues, but there was a lot of red herrings along the way so I could have been mistaken. I was kind of going based off [this doc](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c) which suggests just replacing the `main` field.

Reply:
Didn't seem to cause any harm in my latest tests, so let me go ahead and make that change